word/views.py

```diff
@@ -174,12 +175,71 @@ def create_descriptions(self, request):
         return descriptions
 
 
-class EditView(UpdateView):
-    model = Word
-    fields = [
-        'word', 'ipa', 'desc', 'tags', 'audio', 'wiktionary_link', 'synonyms'
-    ]
-    template_name_suffix = '_update_form'
+def edit_word(request, pk):
+    word = Word.objects.get(id=pk)
+    user_profile = Profile.objects.get(user=request.user)
+    default_variants = UserLanguage.objects.filter(user=user_profile) \
+        .values("language__default_variant", "language_id")
+    user_languages = UserLanguage.objects.filter(user=user_profile)
+    user_moderator = False
+    for user_language in user_languages:

```

Comment:
Without checking, to me it seems that the user just needs to be a moderator of any language, but instead he needs to be a moderator of the language of the word.

Reply:
yes of course!